routing.asciidoc

```diff
@@ -82,304 +99,201 @@ As Wei gets message from the escrow that Bob has deposited the 10 golden coins W
-In that case Bob could just force close the channel and publish his latest version of the commitment transaction.
-As the time lock of the HTLC is not over yet with an onchain success transaction Bob would be able to claim and settle his 15 mBTC as he is the only person who is able to spend the HTLC output in the commitment transaction.
-The other way around meaning Bob and Alice would negotiate a new commitment transaction with the removed HTLC would never be save for Alice.
-If the signatures for the new commitment transaction are exchanged Bob has received the money and could decide not to release the preimage.
+As long as the escrow is trustworthy and faithfully performs its duty, then no party will have their coins stolen in the process.
+The pre-condition to this _route_ working at all, is that all parties in the path already needed to have enough money to satisfy the required series of deposits.
+While this seems like a minor detail we will see in this chapter, this requirement is actually one of the harder issues for Lightning Network nodes.
+As parties are required to deposit the funds used for the payment in escrow, they aren't able to use this money for anything else while the contract is active.
+However, as discussed earlier, they're compensated for this opportunity cost in the forms of earned routing fees when they successfully forward the payment.
 
-[NOTE]
-====
-Isn't it remarkable that even though the process of exchanging funds for an preimage seems to be happening concurrently at the same moment in time in reality it is actually happening one step after another but in the right order.
-====
+In the next section, we'll dive into the details of the "HTLC", exploring the properties that allow it to implement the concept of a conditional chained end to end secure payment _without_ involving the 3rd party escrow.
+You'll see how the Bitcoin network can be seen as a stand-in for a trusted third party to ensure proper HTLC contract execution. 
+After that, we'll explore how a user is able to use an HTLC to "route" a payment through the network securely.
+In the Lightning Network today, we use a technique called source-based onion routing, though it's also possible to route payment with alternative techniques.
+Finally, we'll dive into the precise details concerning the exact mechanics of forward, settle ling, and cancelling HTLCs in the network.
+
+= Hash Time Locked Contracts as a Conditional Chained End to End Secure Payment =
+
+Our example in the prior section using "golden coins", was intended to lay same base intuition which we'll leverage in this section to explain how HTLCs work in practice.
+HTLC is actually an acronym that stands for "Hash Time-Locked Contracts". 
+A HTLC is a _specific_ instantiation of a Conditional Chained End to End Secure Payment (CCESP, don't use this acronym?).
+As we'll see in the later chapters, given a set of adequate cryptographic constructs, many other instantiations are possible as well.
```

Comment:
Would it make sense to give the coinbase transaction as an example as it depends on the condition that it solves the hash puzzle given the current difficulty. where should I put comments like this?

Reply:
Hmm, not that's not the type of cryptographic construction I'm referring to here.